README.md

```diff
@@ -58,6 +58,8 @@ Examples:
 
 | Helper Name  | Example Use In Templates | Example Use in File/Folder Names | Sample Result          |
 |--------------|--------------------------|----------------------------------|------------------------|
+| upperCase    | {{upperCase name}}       | \_\_upperCase_name\_\_               | THISISUPPERCASE        |
+| lowerCase    | {{lowerCase name}}       | \_\_lowerCase_name\_\_               | thisislowercase        |
```

Comment:
Just confirming that if the input contains spaces (example : `Test Action`) we are  expecting the upperCase and lowerCase results to also contain spaces  (ie `test action` and `TEST ACTION`) correct?